exports/exports_test.go

```diff
@@ -537,6 +537,23 @@ var _ = Describe("The public API", func() {
 		Expect(rr.Code).To(Equal(http.StatusBadRequest))
 		Expect(rr.Body.String()).To(ContainSubstring("invalid or missing payload format"))
 	})
+	
+	It("returns the appropriate error if an application or resource is incorrect ", func() {
+		router := setupTest(mockRequestApplicationResources)
+
+		rr := httptest.NewRecorder()
+
+		req := createExportRequest(
+			"Test Export Request",
+			"json",
+			"",
+			`{"application":"exampleApp", "resource":"exampleResource"}`,
+		)
+		AddDebugUserIdentity(req)
+		router.ServeHTTP(rr, req)
+		Expect(rr.Code).To(Equal(http.StatusNotAcceptable))
+		Expect(rr.Body.String()).To(ContainSubstring("Payload does not match Configured Exports"))
+	})
```

Comment:
This test looks good!

It would good to add another version of this test that uses a valid app, but an invalid resource.  You could modify this test to be a table driven test like this:  https://github.com/RedHatInsights/export-service-go/actions/runs/9764633574/job/26953420583?pr=150

The syntax is a bit hard to follow, so reach out if you need a hand!